tensorflow/core/kernels/reduction_ops_mean.cc

```diff
@@ -51,8 +51,10 @@ TF_CALL_NUMBER_TYPES(REGISTER_CPU_KERNELS);
                           ReductionOp<GPUDevice, type, int64,               \
                                       Eigen::internal::MeanReducer<type>>);
 TF_CALL_GPU_NUMBER_TYPES(REGISTER_GPU_KERNELS);
+#if GOOGLE_CUDA
 TF_CALL_complex64(REGISTER_GPU_KERNELS);
```

Comment:
either leave comment here, or change `core_kernels.md` so we can revisit later?

Reply:
The are marked as not done in core_kernels.md .